{"name": "Delete pet with invalid id", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 400\n +  where 404 = <Response [404]>.status_code", "trace": "self = <test_api.TestApi object at 0x00000164D03EA870>, api_client = <utils.api_client.ApiClient object at 0x00000164CF4777D0>\n\n    @allure.title(\"Delete pet with invalid id\")\n    def test_delete_pet_400(self, api_client):\n        response = api_client.delete(\"/pet/p\")\n        print(response)\n        with allure.step(f\"Check status {response.status_code}\"):\n>           assert response.status_code == 400\nE           assert 404 == 400\nE            +  where 404 = <Response [404]>.status_code\n\ntests\\test_api.py:207: AssertionError"}, "steps": [{"name": "Delete request to '/pet/p'", "status": "passed", "parameters": [{"name": "path", "value": "'/pet/p'"}], "start": 1704888928516, "stop": 1704888929646}, {"name": "Check status 404", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 400\n +  where 404 = <Response [404]>.status_code\n", "trace": "  File \"D:\\Pycharm_Projects\\api-testing-exam\\tests\\test_api.py\", line 207, in test_delete_pet_400\n    assert response.status_code == 400\n"}, "start": 1704888929646, "stop": 1704888929646}], "attachments": [{"name": "stdout", "source": "2b8de62e-5795-42d8-8989-3702782d5995-attachment.txt", "type": "text/plain"}], "start": 1704888928515, "stop": 1704888929646, "uuid": "7f4a193a-0bc5-47d6-8f9d-b535c056a255", "historyId": "9f8c5975cc427615cef607cf3818b5f5", "testCaseId": "9f8c5975cc427615cef607cf3818b5f5", "fullName": "tests.test_api.TestApi#test_delete_pet_400", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "DESKTOP-CPFF6BR"}, {"name": "thread", "value": "27720-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}